src/modules/jobs/models/tech-load.model.ts

```diff
@@ -0,0 +1,399 @@
+export class TechLoad {
+
+  public jobs: JobData[];
```

Comment:
do these need to be public? Or even global?

Reply:
Hmm. It's not used here, but "technically" is used on the front end. Not sure how it packages the data to be sent, but I assume it takes every public variable of the class and then converts it into a generic object. prob json.stringify. I'll have to test, but I'm guessing if I make all of this data private or not global than it will not be sent to the front end.

Reply:
I made them private except for jobs, and changed how I pass the job data to the front end. I was sending this entire class object to the front end, which includes all public and private variables. I instead made everything private except jobs, and then just send the array of jobs to the front end. I adjusted the front end service file accordingly in the sister PR